sdk/template/Azure.Template/src/Models/SecretBundle.Serialization.cs

```diff
@@ -77,5 +80,55 @@ internal static SecretBundle FromResponse(Response response)
             using var document = JsonDocument.Parse(response.Content);
             return DeserializeSecretBundle(document.RootElement);
         }
+
+        /// <summary>
+        /// TBD.
+        /// </summary>
+        /// <param name="secretBundle"></param>
+        public static implicit operator RequestContent(SecretBundle secretBundle)
+        {
+            if (secretBundle == null)
+            {
+                return null;
+            }
+
+            return RequestContentExtensions.Create(secretBundle, ModelReaderWriterOptions.DefaultWireOptions);
+        }
+
+        // TODO: Are we happy with taking a dependency on STJ types in client libraries per our prior analyzer AZC0014?
```

Comment:
We should consider how widely we're prepared to suppress AZC0014 when we move to the new ModelReaderWriter APIs.  We introduced `AZC0014` to prevent client libraries from exposing an implementation detail in GA APIs, and now we are requiring them to do exactly that.